runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java

```diff
@@ -148,4 +149,26 @@ default boolean confirmKicks()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "showIgnores",
+		name = "Marks players who are on ignore list",
```

Comment:
Ideally the config names should be fairly short since it is what shows in the panel, and then have a longer description. maybe `Mark ignored players` or something.